### PR TITLE
[1.18] Fix CI

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind failing-test

#### What this PR does / why we need it:
Fixing the CircleCI pipeline for this branch.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,6 +92,7 @@ else
 GCFLAGS = -gcflags '-N -l'
 endif
 
+VERSION ?= $(shell $(GO_RUN) ./scripts/latest-version)
```
Comment (haircommander) [inline Makefile:95]: is this used anywhere?

Comment (saschagrunert) [reply Makefile:95]: Yes in the release notes script for example.

**Comment by haircommander** [comment]
I'm not a huge fan of partially using this script to scrape the version... the hardcoded version and the dynamic one could differ...
Unfortunately, we're stuck between a rock and a hard place here.

Either we:
- take the approach of this PR (use both hardcoded and dynamic)
- hardcode the VERSION in the makefile (fixing the release notes scripts)
    - one would think this would do what we want, but I think we would have similar problems pulling this into openshift that caused the revert in the first place, I have to check the spec file
- hardcode in Makefile AND in internal/version/version.go
    - which could also be prone to skew
- parse the version from the output of crio --version for the release notes
    - seems needlessly complicated...

**Comment by saschagrunert** [comment]
> I'm not a huge fan of partially using this script to scrape the version... the hardcoded version and the dynamic one could differ...
> Unfortunately, we're stuck between a rock and a hard place here.
> 
> Either we:
> 
> * take the approach of this PR (use both hardcoded and dynamic)
> * hardcode the VERSION in the makefile (fixing the release notes scripts)
>   
>   * one would think this would do what we want, but I think we would have similar problems pulling this into openshift that caused the revert in the first place, I have to check the spec file
> * hardcode in Makefile AND in internal/version/version.go
>   
>   * which could also be prone to skew
> * parse the version from the output of crio --version for the release notes
>   
>   * seems needlessly complicated...

I'll check if I can fix this pipeline by relying on the version in the internal/version package.

**Comment by haircommander** [comment]
/lgtm

thanks @saschagrunert

**Pull request merged**